contracts/Trader.sol

```diff
@@ -150,7 +160,7 @@ contract Trader is ITrader {
      * @param order the limit order being hashed
      * @return an EIP712 compliant hash (with headers) of the limit order
      */
-    function hashOrder(Types.Order memory order) public view override returns (bytes32) {
+    function hashOrder(Perpetuals.Order memory order) public view override returns (bytes32) {
```

Comment:
Just raising that this function is pretty much the same as `Perpetual.orderId(order)`, however it uses `abi.encodePacked`. I know this is used for comparing EIP712 signatures so may not be worth using the `orderId` function, but leaving a comment to just raise the point.

I realise this is used by the EIP712 spec for signing so maybe it shouldn't and this should be kept as is here, either in case we change the ID implementation or if we don't want to use `abi.encodePacked` for the order id just worth raising.

Reply:
Does EIP-712 mandate packed encoding?

Reply:
@sporejack due to how the ethSignTypedData RPC call works, I am fairly certain it needs data to be encoded in packed form, so yeah it does.

Thats the main reason I would be pro keeping this function. Just wanted to raise it so we were all aware

Reply:
@raymogg If it *is* mandated then yes, I'd agree in that we do need this function to remain.